core/modules/user/css/user.css

```
@@ -108,6 +108,7 @@
 .password-strength.weak .indicator {
   background-color: #ff0000;
 }
+.password-strength.tooWeak .password-strength-text,
```

Weird to see capital letters in CSS. I believe that it's not allowed by our coding standards. Perhaps this instead?:

```suggestion
.password-strength.too-weak .password-strength-text,
```